src/common.c

```diff
@@ -79,6 +79,20 @@ struct ofi_common_locks common_locks = {
 	.util_fabric_lock = PTHREAD_MUTEX_INITIALIZER,
 };
 
+#if !HAVE_CLOCK_GETTIME
+int clock_gettime(clockid_t clk_id, struct timespec *tp) {
+	int retval;
+	struct timeval tv;
+
+	retval = gettimeofday(&tv, NULL);
+
+	tp->tv_sec = tv.tv_sec;
+	tp->tv_nsec = tv.tv_usec * 1000;
+
+	return retval;
+}
+#endif /* !HAVE_CLOCK_GETTIME */
```

Comment:
This works for non-windows platforms.  For windows, we need:
```
static inline
int clock_gettime(int which_clock, struct timespec *spec)
{
	__int64 wintime;

	GetSystemTimeAsFileTime((FILETIME*)&wintime);
	wintime -= win2unix_epoch;

	spec->tv_sec = wintime / file2unix_time;
	spec->tv_nsec = wintime % file2unix_time * 100;

	return 0;
}
```
Search fabtests for clock_gettime.  This should be in windows/osd.h

Reply:
@shefty Thanks, it is fixed for WinOS now.